src/features/lightspeed/base.ts

```diff
@@ -106,6 +127,50 @@ export class LightSpeedManager {
     this.setCustomWhenClauseContext();
   }
 
+  /* Update the state of the local user and show the Connect button if needed */
+  public async refreshUserInfo() {
+    console.log(
+      "[LightSpeedManager] Sending request for logged-in user info...",
+    );
+
+    try {
+      const data: UserResponse = await this.apiInstance.getData(
+        `${getBaseUri(this.settingsManager)}${LIGHTSPEED_ME_AUTH_URL}`,
+      );
+      console.log(`refreshUserInfo: ${data}`);
+      console.log(`userIsconnected!, lets'refresh`);
+      this.lightSpeedAuthenticationProvider.userIsConnected = true;
+      this.lightspeedExplorerProvider.refreshWebView();
+      this.statusBarProvider.updateLightSpeedStatusbar();
+      // NOTE: This lightspeedConnectReady doesn't seem to be actually used
```

Comment:
This `lightspeedConnectReady` context  was added with #1076, which was a timing issue caused because the initialization (`activate`) takes longer than usual in certain situation.  We need to make sure if the issue that was resolved by #1076 does not occur after this change.